network/src/main/java/org/jboss/as/network/OutboundSocketBinding.java

```diff
@@ -160,9 +160,6 @@ public String getUnresolvedDestinationAddress() {
      * @throws UnknownHostException If the destination address cannot be resolved
      */
     public synchronized InetAddress getResolvedDestinationAddress() throws UnknownHostException {
-        if (this.resolvedDestinationAddress != null) {
-            return this.resolvedDestinationAddress;
-        }
```

Comment:
This should be kept so we can use the resolvedDestinationAddress passed in to the constructor.